### PR TITLE
Add extra XCB dependencies required to compile Qt 5.15

### PR DESCRIPTION
**Pull request by andfoy**
Please see the repo readme for directions on how to make PRs on this repo.

Checklist:

- [x] if you have added a CDT, it appears in the `cdt_slugs.yaml` file and
  you have rerun the script `python gen_cdt_recipes.py`.
- [ ] if you have changed the CDT generator code (`rpm.py`), you have bumped
  the build number in `conda_build_config.yaml` and have remade all of the
  recipes via running `python gen_cdt_recipes.py --force`
- [ ] if you have added a custom CDT recipe, you have added the name of the CDT
  with `custom: true` in the `cdt_slugs.yaml` file.
- [ ] all CDT recipes have build number set by `{{ cdt_build_number }}` for
  old-style/legacy CDTs or `{{ cdt_build_number|int + 1000 }}` for new-style CDTs
- [ ] if you see a warning about a CDT not having a license, you have added the
  `license_file` key in the `cdt_slugs.yaml` file with the path to the appropriate
  license in `licenses/`

**NOTE: If you make any changes to `cd_slugs.yaml`, you need to reun the generator code
via `python gen_cdt_recipes.py`.**

### CODE REVIEW

File: cdt_slugs.yaml [hunk 1]
```diff
@@ -597,6 +603,24 @@ webkitgtk-devel:
...
+  license_file: licenses/xcb-util-license
+xcb-util-image-devel:
+  custom: false
+  license_file: licenses/xcb-util-image-license
+xcb-util-keysyms:
+  custom: false
+  license_file: licenses/xcb-util-keysyms-license
+xcb-util-keysyms-devel:
+  custom: false
+  license_file: licenses/xcb-util-keysyms-license
```
Comment (andfoy) [inline cdt_slugs.yaml:617]: @isuruf, the CI is complaining about the license of `xcb-util-keysyms`, however it is listed here. Am I doing something wrong here?

**Comment by beckermr** [comment]
There seem to be changes to existing CDTs in this PR that should not have happened.

**Comment by andfoy** [comment]
@beckermr, I was wondering why there were so many changes on other CDTs, I just followed the instructions listed on the README over the changes of 14a14251e6d435ba959c3861f0ac8cd4fde603c1

**Comment by beckermr** [comment]
Yeah. Don't use the --fast option and make sure your branch is up to date with master.

**Comment by beckermr** [comment]
I will try it locally too.

**Comment by andfoy** [comment]
Should I use the `--force` option?

**Pull request closed without merging**